invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/records/search/filters/DeletionStatusFilter.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * // This file is part of Invenio-App-Rdm
+ * // Copyright (C) 2023 CERN.
+ * //
+ * // Invenio-App-Rdm is free software; you can redistribute it and/or modify it
+ * // under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import { i18next } from "@translations/invenio_requests/i18next";
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { withState } from "react-searchkit";
+import { Button } from "semantic-ui-react";
+
+class DeletionStatusFilterComponent extends Component {
```

Comment:
I don't fully understand why we are re-creating filter widgets, instead of using React-SearchKit out of the box, but I trust that you did not have any other solution with existing APIs from RSK.